incubator/postfix/templates/secrets.yaml

```diff
@@ -9,6 +9,11 @@ metadata:
     heritage: "{{ .Release.Service }}"
 type: Opaque
 data:
-  smtp_username: {{ trim .Values.smtp.username | b64enc }}
   smtp_password : {{ trim .Values.smtp.password | b64enc }}
   smtp_domain: {{ trim .Values.smtp.domain | b64enc }}
```

Comment:
@goruha this is what I meant =)

https://github.com/cloudposse/charts/pull/73/commits/a93ed03d1a88cea74668a556d6a753de0d34d82c